lib/hammer_cli_katello/repository.rb

```diff
@@ -220,6 +221,8 @@ def execute
           File.open(file_path, 'rb') { |file| upload_file file }
```

Comment:
I thought about that, but actually thought this might be better.  Doing it the way you suggest would mean that if an error occured part way through uploading the files, none of the files would be uploaded (because the import step wouldn't be called).  This way everything up to the error is fully uploaded.   I might could see arguments either way but i think i prefer this method.